### ISSUE TITLE
Cookie-signature creating some confusion

### ISSUE THREAD

**Issue by matthewdking** [foundersandcoders/ws-session-management#26]
There was some confusion when the workshop was ran this time in Nazareth due to the cookie signature npm package being mentioned in exercise 1. The main confusion came about because students then thought they should be using this package to complete the exercise but it is purely there as a resource to be looked at. Students were even more confused as the docs for cookie signature aren't the best.

I don't want to suggest removing it as a resource because some students specifically mentioned it being useful, however maybe being more clear about using bcrypt specifically for the exercise?

**Comment by jcharnley** [comment]
I agree exercise 1 is confusing. There should be a link or example of the createHmac syntax or link in the area within that exercise, instead of it being up at the top. As you have said  cookie0signature npm links are not needed to complete the exercise.
